.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: tests
+on:
+  push:
+    branches:
+    - master
+    - dev
+    tags:
+    - v*
+  pull_request:
+    branches:
+    - dev
+jobs:
+  test:
+    name: python-${{ matrix.python }} (${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 40  # default is 360
+    strategy:
+      matrix:
+        python:
+        - 3.6
+        - 3.7
+        os:
+        - ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python }}
+    - name: Install Poetry
+      run: pip -q --no-input install poetry
+    - name: Install Poetry dependencies
+      run: poetry install -n -E client --no-root
```

Comment:
`-E client` is for adding the `client` extras, I added this on `hathorlib` since it has that extra, I don't think it should be here

Reply:
Fixed. Thanks!